### ISSUE TITLE
Getting "error occurred while generating"

### ISSUE THREAD

**Issue by deantaylormax** [bionic-gpt/bionic-gpt#134]
Tried installing the tool twice following website instructions.  No errors on install.  However, any prompt generates "error occurred while generating". 

The terminal had this potentially relevant error message:  Ideas?

lawstata-unstructured-1    | 2023-12-02 02:40:52,476 unstructured_api ERROR Json schema does not match the Unstructured schema
lawstata-unstructured-1    | 2023-12-02 02:40:52,477 172.18.0.8:60704 POST /general/v0/general HTTP/1.1 - 400 Bad Request
lawstata-pipeline-job-1    | Error: reqwest::Error { kind: Decode, source: Error("invalid type: map, expected a sequence", line: 1, column: 0) }
lawstata-pipeline-job-1 exited with code 1
lawstata-app-1             | 2023-12-02T02:42:37.447797Z  INFO axum_server::prompt: prompt.name="Default (Exclude All Datasets)"
lawstata-app-1             | 2023-12-02T02:42:37.449312Z  INFO axum_server::prompt: Retrieved 0 chunks
lawstata-app-1             | 2023-12-02T02:42:37.458280Z  INFO axum_server::prompt: Retrieved 0 history items
lawstata-app-1             | 2023-12-02T02:42:37.738348Z ERROR axum_server::errors: response="status = 422 Unprocessable Entity, message = error trying to connect: tcp connect error: Connection refused (os error 111)"
lawstata-app-1             | 2023-12-02T02:42:37.785068Z  INFO axum_server::prompt: prompt.name="Default (Exclude All Datasets)"
lawstata-app-1             | 2023-12-02T02:42:37.785093Z  INFO axum_server::prompt: Retrieved 0 chunks
lawstata-app-1             | 2023-12-02T02:42:37.786561Z  INFO axum_server::prompt: Retrieved 0 history items
lawstata-app-1             | 2023-12-02T02:42:38.013440Z ERROR axum_server::errors: response="status = 422 Unprocessable Entity, message = error trying to connect: tcp connect error: Connection refused (os error 111)"
lawstata-app-1             | 2023-12-02T02:42:38.014442Z  INFO axum_server::prompt: prompt.name="Default (Exclude All Datasets)"
lawstata-app-1             | 2023-12-02T02:42:38.014470Z  INFO axum_server::prompt: Retrieved 0 chunks
lawstata-app-1             | 2023-12-02T02:42:38.015346Z  INFO axum_server::prompt: Retrieved 0 history items
lawstata-app-1             | 2023-12-02T02:42:38.229327Z ERROR axum_server::errors: response="status = 422 Unprocessable Entity, message = error trying to connect: tcp connect error: Connection refused (os error 111)"
lawstata-app-1             | 2023-12-02T02:42:38.298760Z  INFO axum_server::prompt: prompt.name="Default (Exclude All Datasets)"
lawstata-app-1             | 2023-12-02T02:42:38.298794Z  INFO axum_server::prompt: Retrieved 0 chunks
lawstata-app-1             | 2023-12-02T02:42:38.299944Z  INFO axum_server::prompt: Retrieved 0 history items
lawstata-app-1             | 2023-12-02T02:42:38.526802Z ERROR axum_server::errors: response="status = 422 Unprocessable Entity, message = error trying to connect: tcp connect error: Connection refused (os error 111)"

**Comment by AnselmJeong** [comment]
Me Too

**Comment by ianpurton** [comment]
@amarjeet and @AnselmJeong are you guys on Apple silicon OSX?

Can someone kindly do a `docker ps` and show me which services are running. Thanks

**Comment by ianpurton** [comment]
Also we did have an issue https://github.com/bionic-gpt/bionic-gpt/issues/107 which I thought was resolved.

But that might help.

**Comment by d0rc** [comment]
Yes, there is a solution in the discussion there. The idea is one can allow app service to go to primary host.

**Comment by mcbuckeye** [comment]
Getting a similar error on my M1 Mac:

bionic-gpt-app-1 | 2023-12-03T23:07:08.320061Z ERROR axum_server::errors: response="status = 422 Unprocessable Entity, message = error trying to connect: tcp connect error: Connection refused (os error 111)"

**Comment by ianpurton** [comment]
Updated docs https://bionic-gpt.com/docs/running-locally/apple-silicon/
